viewer/src/components/Layout.tsx

```diff
@@ -46,6 +48,19 @@ class Layout extends React.PureComponent<Props> {
       <div>
         <Navbar />
 
+
+        { location.pathname === '/' ? 
```

Comment:
Why isn't this just part of `<Home/>`? In any case, I'd prefer we did our routing consistently with `<Route/>`s like https://github.com/pi-base/server/blob/ede6b2c331e2c75a7c7e9ce9a925cbf1e42cb84d/viewer/src/routes.tsx#L21

Reply:
Oh! It's because you don't want it in the `.container`. :+1:

Reply:
Yeah, although I opened #5 to get the best of both worlds.